runtime/common/src/lib.rs

```diff
@@ -277,6 +277,12 @@ parameter_types! {
 	pub const TechnicalMaxMembers: u32 = 10;
 }
 
+// pallet vesting
+parameter_types! {
+	pub MinVestedTransfer: Balance = 1_000 * HDX;
```

Comment:
Is this per account ? Isnt it too high ? 

Eg. recent crowdloan bonus rewards are lower than this if contributed 1KSM for example.

Reply:
It is per `vesting_transfer`, so you can't vest less than this amount.

Reply:
why don't we just set it to 0?

Reply:
what about setting it to `NATIVE_EXISTENTIAL_DEPOSIT`?

Reply:
yes makes sense too

Reply:
Oh, we can't use it because we don't have the duster pallet there.

Reply:
set to 0 for now, it will have to be increased if we allowed vested transfer to anyone

Reply:
We should definitely set it to existential deposit and we have that here. Just extract it to const.